e2e/system/files_to_volumes_cm_test.go

```diff
@@ -11,43 +11,6 @@ import (
 	"github.com/celestiaorg/knuu/pkg/instance"
 )
 
-// TestOneVolumeNoFiles tests the scenario where we have one volume and no files.
-// the initContainer command that it generates looks like:
-// no initContainer command, as there is no volumes, nor files.
-func (s *Suite) TestNoVolumesNoFiles() {
```

Comment:
Why did we delete this entire test?

Reply:
@tty47 I have the same question as well. Is it not needed anymore?